minifier.js

```diff
@@ -13,12 +13,20 @@ function minifyHTML(opts) {
             if (typeof callback === 'undefined') {
                 return function (err, html) {
                     if (err) {
-                        console.error(err)
-                        if (opts.displayErrors === true) {
-                            res.send('Rendering error: ' + err.message);
-                        } else {
-                            res.sendStatus(500);
+                        if (typeof opts.errCallback === 'function') //custom error callback specified
+                        {
+                            return opts.errCallback(err, req, res, next);
                         }
+                        else //no custom error callback specified, default
```

Comment:
Could you put the else on the same line as the ending bracket, and the beginning of the else after the keyword, like at line 30?